SparkPost/2web2ui
# FE-854 Codecov Integration

### What Changed
 - Added Codecov integration as the last step (only on success) in travis process
 - Removed remnants of coveralls in repo

### How To Test
 - Navigate to https://codecov.io/gh/SparkPost/2web2ui/commits and see that this branch is there

### To Do
- Add Codecov integration (bot) to this repo so we get the automatic posts
